tools/applicationinsights-web-snippet/src/applicationinsights-web-snippet.ts

```diff
@@ -13,13 +14,28 @@ function webSnippetVersion() {
 }
 
 function getSdkLoaderScript(config: SdkLoaderConfig) {
-    let snippet: string = webSnippetCs;
-    if (config && config.connectionString) {
-        snippet = webSnippetCs.replace("YOUR_CONNECTION_STRING", config.connectionString);
-    } else if (config && config.instrumentationKey) {
-        snippet = webSnippet.replace("InstrumentationKey=INSTRUMENTATION_KEY", config.instrumentationKey);
+    let snippetConfig: ISnippetConfig = {
+        src: config.src? config.src : "https://js.monitor.azure.com/scripts/b/ai.3.gbl.min.js",
+        crossOrigin: config.crossOrigin ? config.crossOrigin : "anonymous",
+        cfg: {},
+        name: config.name ? config.name : "appInsights",
+        ld: config.ld,
+        useXhr: config.useXhr,
+        onInit: config.onInit,
```

Comment:
Can you write a quick test to see if you pass in a function whether it will actually get called?

I don't think it will, as I think this process is just going to populate with a text version of the function....

If it doesn't work, lets just remove it from this version as we are going to have to do something tricky to handle / pass a Promise through this for the same reason as why I think the function isn't going to work.

Reply:
it doesn't work, I will remove onInit for this version. Thanks!